### PR TITLE
suggest knexfile

### PR DESCRIPTION
**Pull request by freben**
OK, this takes a bad situation and makes it only very slightly better. The goal is to be able to run a command to create migration files that are immediately usable, instead of copy-pasting.

Turns out that it works to have a single knexfile in the root after all. But it's quirky. See the comment in the knexfile itself. There's no templating so the stub file has a hardcoded copyright year. Baby steps. I'm not even gonna document this anywhere else for now. Until we make it decidedly better, there's no real use to do so, or to package it up in a way that's usable for others.

### CODE REVIEW

File: .config/knex-migration.stub.js
Comment (freben) [inline .config/knex-migration.stub.js:1]: The diff is not neat, obviously this is not a move but a deletion + addition

**Review by Rugvip — APPROVED** [review]
Nice!-ish :p, it does indeed make things better

I wonder if `.config/knex-migration.stub.js` could be in contrib instead? always nice to avoid more top-level folders and it might actually be useful? As you said you didn't wanna document this any more atm, so happy to merge as is too.

**Comment by freben** [comment]
Yeah the idea with .config seems to be that all kinds of packages are moving toward supporting their configs in there, _instead_ of top level folders. Hopefully this is the first of several usages of it, buuuut ... who knows

**Comment by freben** [comment]
Updated!

**Pull request merged**